From 62c2c1c110f7d569110876108963d118fe20af34 Mon Sep 17 00:00:00 2001
From: Karol Latecki <karol.latecki@intel.com>
Date: Thu, 28 Jan 2021 14:08:31 +0100
Subject: [PATCH] scripts/nvmf_perf: fix pylint E1111 error - assignment on no
 return

Fix assigning variables from functions which return nothing.

Change-Id: Ic54dc0d42e3339cda23970ed347ac6f7bf92e795
Signed-off-by: Karol Latecki <karol.latecki@intel.com>
Reviewed-on: https://review.spdk.io/gerrit/c/spdk/spdk/+/6203
Tested-by: SPDK CI Jenkins <sys_sgci@intel.com>
Reviewed-by: Maciej Wawryk <maciejx.wawryk@intel.com>
Reviewed-by: Tomasz Zawadzki <tomasz.zawadzki@intel.com>
Reviewed-by: Jim Harris <james.r.harris@intel.com>
Community-CI: Mellanox Build Bot
---
 scripts/perf/nvmf/run_nvmf.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/scripts/perf/nvmf/run_nvmf.py b/scripts/perf/nvmf/run_nvmf.py
index 2c5331e86..63179923a 100755
--- a/scripts/perf/nvmf/run_nvmf.py
+++ b/scripts/perf/nvmf/run_nvmf.py
@@ -647,11 +647,11 @@ class SPDKTarget(Target):
         rpc.client.print_dict(rpc.nvmf.nvmf_get_transports(self.client))
 
         if self.null_block:
-            nvme_section = self.spdk_tgt_add_nullblock(self.null_block)
-            subsystems_section = self.spdk_tgt_add_subsystem_conf(self.nic_ips, self.null_block)
+            self.spdk_tgt_add_nullblock(self.null_block)
+            self.spdk_tgt_add_subsystem_conf(self.nic_ips, self.null_block)
         else:
-            nvme_section = self.spdk_tgt_add_nvme_conf()
-            subsystems_section = self.spdk_tgt_add_subsystem_conf(self.nic_ips)
+            self.spdk_tgt_add_nvme_conf()
+            self.spdk_tgt_add_subsystem_conf(self.nic_ips)
         self.log_print("Done configuring SPDK NVMeOF Target")
 
     def spdk_tgt_add_nullblock(self, null_block_count):
-- 
2.37.2.windows.2

